src/libpriv/rpmostree-kernel.c

```diff
@@ -457,7 +457,7 @@ rpmostree_run_dracut (int     rootfs_dfd,
     "#!/usr/bin/bash\n"
     "set -euo pipefail\n"
     "extra_argv=; if (dracut --help; true) | grep -q -e --reproducible; then extra_argv=\"--reproducible --gzip\"; fi\n"
-    "dracut $extra_argv -v --add ostree --tmpdir=/tmp -f /tmp/initramfs.img \"$@\"\n"
+    "mkdir -p /tmp/dracut && dracut $extra_argv -v --add ostree --tmpdir=/tmp/dracut -f /tmp/initramfs.img \"$@\"\n"
```

Comment:
We don't need the `mkdir` though, right? We're already bind mounting there so it will already exist, no?

Reply:
We only bind mount in the compose case; client side builds don't currently.   (Though they probably should).  

(Dracut may also implicitly do the `mkdir`, I haven't looked but it seems clearer if we do it)